operation/fhir-operation-bulkdata/src/test/java/com/ibm/fhir/operation/bulkdata/util/BulkDataExportUtilTest.java

```diff
@@ -337,7 +337,7 @@ public void testCheckAndValidateTypesNull() throws FHIROperationException {
         builder.parameter(parameters);
         Parameters ps = builder.build();
 
-        util.checkAndValidateTypes(ps, null);
+        util.checkAndValidateTypes(ExportType.SYSTEM,ps, null);
```

Comment:
adding the space to all these would be a little nicer to read
```suggestion
        util.checkAndValidateTypes(ExportType.SYSTEM, ps, null);
```